### PR TITLE
Adding DISTRIBUTION_NAME to Promote-Artifacts jenkins job description

### PR DESCRIPTION
**Pull request by Tokesh**
### Description
The job description for Promote Artifacts is not clear enough. I understand that in order to add it, you need to edit the configuration of only this job. If something is wrong - ready to listen to any comments.
(Readme file doesn't related for this PR, but still under review #3506. If everything is fine for merge of this PR, i'm ready to delete this Readme file in next commit)
### Issues Resolved
[#3456]

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: jenkins/promotion/promote-artifacts.jenkinsfile [hunk 1]
```diff
@@ -54,7 +54,7 @@ pipeline {
             }
             steps {
                 script {
-                    currentBuild.description = "${DISTRIBUTION_BUILD_NUMBER} ${DISTRIBUTION_PLATFORM} ${DISTRIBUTION_ARCHITECTURE} ${INPUT_MANIFEST}"
+                    currentBuild.description = "${DISTRIBUTION_BUILD_NUMBER} ${DISTRIBUTION_PLATFORM} ${DISTRIBUTION_ARCHITECTURE} ${INPUT_MANIFEST} ${DISTRIBUTION_NAME}"
```
Comment (peterzhuamazon) [inline jenkins/promotion/promote-artifacts.jenkinsfile:57]: Hi @Tokesh thanks for this tho it is also beneficial to add `${DISTRIBUTION_ARCHITECTURE}` as well.
Please update.

Thanks.

**Review by peterzhuamazon — CHANGES_REQUESTED** [review]

File: jenkins/promotion/promote-artifacts.jenkinsfile (hunk 1 above)
Comment (Tokesh) [reply jenkins/promotion/promote-artifacts.jenkinsfile:57]: > Hi @Tokesh thanks for this tho it is also beneficial to add `${DISTRIBUTION_ARCHITECTURE}` as well. Please update.
> 
> Thanks.

Hello @peterzhuamazon, but it already added. Can you recheck please?

Comment (peterzhuamazon) [reply jenkins/promotion/promote-artifacts.jenkinsfile:57]: Oops, I must have missed that. Thanks.

Comment (peterzhuamazon) [reply jenkins/promotion/promote-artifacts.jenkinsfile:57]: Can you add ${DISTRIBUTION_NAME}" before ${INPUT_MANIFEST} then?

Comment (Tokesh) [reply jenkins/promotion/promote-artifacts.jenkinsfile]: > Can you add ${DISTRIBUTION_NAME}" before ${INPUT_MANIFEST} then?

Of course! Its ready for your check

Comment (peterzhuamazon) [reply jenkins/promotion/promote-artifacts.jenkinsfile]: Thanks, merged.

**Comment by peterzhuamazon** [comment]
@Tokesh your change is now in production Jenkins, see list of descriptions of the runs:
https://build.ci.opensearch.org/job/distribution-promote-artifacts/

Thanks!

**Pull request merged**